packages/devtools_app/lib/src/screens/memory/shared/shared_memory_widgets.dart

```diff
@@ -5,7 +5,7 @@
 import 'package:flutter/widgets.dart';
 
 import '../../../../devtools_app.dart';
```

Comment:
remove the devtools_app import

Reply:
will do in next PR. I really want to avoid conflicts :)